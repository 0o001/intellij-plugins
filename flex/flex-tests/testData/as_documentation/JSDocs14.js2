class C {
  /**
   *  @private
   *  This method allows access to native implementation
   */
  mx_internal final function $remove<caret>Child(child:DisplayObject):DisplayObject;
}