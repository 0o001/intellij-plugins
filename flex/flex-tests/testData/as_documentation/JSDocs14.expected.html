<div class='definition'><pre>private<br>C.$removeChild(<br>    child: <a href="psi_element://DisplayObject">DisplayObject</a>): <a href="psi_element://DisplayObject">DisplayObject</a></pre></div><div class='content'><p>This method allows access to native implementation</div>